src/MomentTz.re

```diff
@@ -257,6 +258,7 @@ module Moment = {
   [@bs.send.pipe: t] external year: int = "";
   [@bs.send.pipe: t] external weekday: int = "";
   [@bs.send.pipe: t] external tz: Js.undefined(string) = "";
+  [@bs.send.pipe: t] external utcOffset: int = "";
```

Comment:
I think all the other changes except this one are unintentional. Suggest tweaking so this is the only change.